### PR TITLE
Add ability to set a custom aria-label value for the actions menu button

### PR DESCRIPTION
**Pull request by ma12-co**
(This is strictly what I need for talk)

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/Actions/Actions.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 	<!-- if only one action, check if we need to bind to click or not -->
 	<element v-if="isValidSingleAction && !forceMenu"
 		v-tooltip.auto="firstAction.text"
-		:aria-label="firstAction.text"
+		:aria-label=" !!ariaLabel ? ariaLabel : firstAction.text"
```
Comment (nickvergessen) [inline src/components/Actions/Actions.vue]:
```suggestion
		:aria-label="ariaLabel || firstAction.text"
```

Comment (ma12-co) [reply src/components/Actions/Actions.vue]: Are you sure??? Wouldn't that return true or false??

Comment (nickvergessen) [reply src/components/Actions/Actions.vue]: Open your browsers console and do:
```
var a = ''
var b = 'bar'
a || b

> "bar"
```

File: src/components/ActionButton/ActionButton.vue [hunk 2]
```diff
@@ -44,7 +44,9 @@ If you're using a long text you can specify a title
 
 <template>
 	<li :class="{ 'action--disabled': disabled }">
-		<button class="action-button"
+		<button
+			class="action-button"
+			:aria-label="ariaLabel"
```
Comment (skjnldsv) [inline src/components/ActionButton/ActionButton.vue]: I guess this could even get to the ActionTextMixin ?
And be allowed in all ActionButton/Link/Router?

Comment (ma12-co) [reply src/components/ActionButton/ActionButton.vue]: You mean the prop??

Comment (skjnldsv) [reply src/components/ActionButton/ActionButton.vue]: prop yes :)
But also to the other templates then :thinking:

Comment (ma12-co) [reply src/components/ActionButton/ActionButton.vue]: Only link and router?

File: src/components/Actions/Actions.vue [hunk 3]
```diff
@@ -264,6 +263,7 @@ export default {
 				if (tag === 'ActionButton') {
 					return {
 						is: 'button',
+						ariaLabel: this.firstAction['ariaLabel'],
```
Comment (skjnldsv) [inline src/components/Actions/Actions.vue]: this is only for ActionButton

Comment (skjnldsv) [reply src/components/Actions/Actions.vue]: Also 
```suggestion
						ariaLabel: this.firstAction.ariaLabel,
```

File: src/components/ActionButton/ActionButton.vue (hunk 2 above)
Comment (skjnldsv) [reply src/components/ActionButton/ActionButton.vue]: Well, All the ones that are allowed as single action, so ActionLink, ActionRouter & ActionButton
https://github.com/nextcloud/nextcloud-vue/blob/0f1a4f317763efeb8beb0cf742245c554b731f6a/src/components/Actions/Actions.vue#L249

File: src/components/Actions/Actions.vue (hunk 3 above)
Comment (skjnldsv) [reply src/components/Actions/Actions.vue]: Did you pushed @ma12-co ? :p

Comment (skjnldsv) [reply src/components/Actions/Actions.vue]: This is still happening here

Comment (ma12-co) [reply src/components/Actions/Actions.vue]: uouups :)

**Pull request merged**